logger.go

```diff
@@ -46,13 +47,19 @@ var defaultLogger = func(lvl logging.Level, s string, args ...interface{}) {
 	if Color {
 		s = color.YellowString(s)
 	}
-	defaultStdLogger.Println(s)
+	if standardLogger != nil {
+		standardLogger.Println(s)
+	}
```

Comment:
the behavior could be somewhat odd when if `standardLogger` is nil for any reason.